src/java.base/share/conf/security/java.security

```diff
@@ -157,9 +157,35 @@ RestrictedSecurity.NSS.140-2.securerandom.algorithm = PKCS11
 #endif
 
 #if defined aix-ppc || defined linux-ppc || defined linux-s390 || defined linux-x86 || defined windows
+#
+# Strict RestrictedSecurity mode profile for FIPS 140-3. This policy represents only allowable
+# approved cryptography in the OpenJCEPlusFIPS provider along with other non cryptographic algorithms
```

Comment:
Missing hyphen in "non-cryptographic", here and on line 315.

Reply:
Please also fix line 315.

Reply:
Done.